.circleci/config.yml

```diff
@@ -45,7 +37,7 @@ jobs:
             - "/go/bin"
 workflows:
   version: 2
-  formatting:
+  test:
```

Comment:
should the workflow be named the same as one of the step? 🤔

Reply:
Well, I think it's an improvement over "formatting", but I'm not sold on it beyond that :)

Reply:
haha, true to that :)